pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -432,6 +432,12 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication {
     {
         address underlyingToken = wrappedToken.asset();
 
+        if (underlyingToken == address(0)) {
+            // Should never happen, but a malicious wrapper could return the zero address and cause the buffer
+            // initialization code to run more than once.
+            revert InvalidUnderlyingTokenAsset();
+        }
```

Comment:
This can only happen when `_bufferAssets[wrappedToken]` is not initialized, but I think it's fine to have it here. Even more clear that `address(0)` asset is not accepted.